curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md

```diff
@@ -7,9 +7,16 @@ dashedName: step-6
 
 # --description--
 
-HTML elements are often nested within other HTML elements. In the previous step you nested the `h2` element, comment and `p` element within the `main` element. A nested element is a child of its parent element.
+In the previous step you nested the `h2` element, comment, and `p` element within the `main` element. A nested element is a child of its parent element and, for readability, should be indented two more spaces than its parent like this:
```

Comment:
I think it might be worth keeping the "nested element being a child of the parent" bit.

> In the previous step, you nested the `h2` element, comment, and `p` element within the `main` element. A nested element is a child of its parent element. It should be indented two more spaces than its parent element to improve readability, like this:

Reply:
Fair, lets do that one